CHANGES.md

```diff
@@ -1,6 +1,11 @@
 ## Changes in 0.6.0. dev
 
 - xcube-cci can now get the time ranges for satellite-orbit-frequency datasets available via opensearch 
+-Descriptions of variables and dimensions are different when data is normalized. 
+In case `normalize_data` is set to True, dimensions will be normalized to `lat`, `lon`, `time` and possibly additional 
```

Comment:
I would add: 

In case `normalize_data` is set to True, dimensions will be normalized to `lat`, `lon`, `time` and possibly additional 
dimensions. E.g. like this: 
 ```
from xcube_cci.dataaccess import CciOdpDataStore
store = CciOdpDataStore(normalize_data=True)
```

Reply:
Yes. Or you can pass this as a parameter to the opener:
from xcube_cci.dataaccess import CciOdpDataOpener
opener = CciOdpDataOpener(normalize_data=True)

Reply:
Nice, can you add that to the changelog? :)

Reply:
Happy now? ;)

Reply:
Very happy, thank you :)